gyp_xwalk

```diff
@@ -354,6 +354,18 @@ if __name__ == '__main__':
     args.append('-Dnotifications=1')
     args.append('-Drelease_unwind_tables=0')
 
+  parameter = '-Dverify_xwalk_apk='
```

Comment:
I understand and agree with what you said, but I don't see how this is related to Debug vs Release. Our bots and pretty much anyone builds in release mode, so if we enabled `verify_xwalk_apk` by default based on build type Crosswalk would break in most cases. I think you just need to make this opt-in, and then we enable it only when building official beta/stable releases.